capstone/src/main/java/com/google/sps/servlets/ServeImageServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Serves the image file corresponding to the given blob key */
+@WebServlet("/get-image")
+public class ServeImageServlet extends HttpServlet {
```

Comment:
Is this different from the servlet I made?

Reply:
It's the same but I don't have the one you made in my local yet so I had to make it again. I'll delete it once I git pull later and just have one in the general servlets folder!